src/SmartFormat/Extensions/LocalizationFormatter.cs

```diff
@@ -68,6 +68,14 @@ public bool TryEvaluateFormat(IFormattingInfo formattingInfo)
         // Get the localized string
         var localized = LocalizationProvider!.GetString(formattingInfo.Format!.RawText, cultureInfo);
 
+        // Try formatting if localized string was not found, but a format has nested items
+        if (localized is null && formattingInfo.Format!.HasNested)
+        {
+            var formatted = _formatter!.Format(formattingInfo.Format!.RawText, formattingInfo.CurrentValue);
```

Comment:
This is quite expensive, because the string that has already been parsed will be parsed again, and will thus create a new `Format` instance.

Reply:
@axunonb I have changed the implementation to avoid double parsing, please take look and check if the modified method is ok

Reply:
Very nice!